src/kirby/components/empty-state/empty-state.component.scss

```
@@ -0,0 +1,31 @@
+@import "../../scss/web-imports";
+
+article {
+  width: 330px;
```

Consider changing to max-width:
```suggestion
  max-width: 330px;
```